readme.md

```diff
@@ -233,6 +433,7 @@ I made this repo to be a more inclusive  version of [sindresorhus's awesome list
 - [Preact](https://github.com/preactjs/awesome-preact#readme) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources#readme)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs#readme) - Framework for server-rendered React apps.
+- [Remix](https://github.com/rjoydip/awesome-remix#readme) - Remix is a full stack web framework that lets you focus on the user interface and work back through web standards to deliver a fast, slick, and resilient user experience..
```

Comment:
**@-sindresorhus** says: The description is a bit too long. And it doesn't need to mention `Remix`.